locale_cs/nbproject/build-impl.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Maybe, we should add the license header?
CC: @matthiasblaesing

Reply:
There is a comment on lin 2-4 in this file 
```
<!--
*** GENERATED FROM project.xml - DO NOT EDIT  ***
***         EDIT ../build.xml INSTEAD         ***
-->
```
I am not sure if we can add another comment with licence header, it will be overrided probably when generated from project.xml file

Reply:
See: https://github.com/apache/netbeans/blob/master/nb/updatecenters/extras/nbjavac.impl/nbproject/build-impl.xml

Reply:
Ok, header added similar to your example.